From af0873a02bb0014b1c80e4665eae9234b56fcfe3 Mon Sep 17 00:00:00 2001
From: Zhang Jiejing <jiejing.zhang@freescale.com>
Date: Mon, 27 Aug 2012 10:20:36 +0800
Subject: [PATCH 049/602] ENGR00221447 imx6: not remount as ro when error ,
 panic when meet error.

Fix the typo of last commit.

Signed-off-by Zhang Jiejing <jiejing.zhang@freescale.com>
---
 imx6/sabresd/init.rc | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/imx6/sabresd/init.rc b/imx6/sabresd/init.rc
index 2977d48..99d709e 100755
--- a/imx6/sabresd/init.rc
+++ b/imx6/sabresd/init.rc
@@ -130,7 +130,6 @@ on fs
 # mount ext4 partitions
     mount ext4 /dev/block/mmcblk0p5 /system
     mount ext4 /dev/block/mmcblk0p5 /system ro remount
-    mount ext4 /dev/block/mmcblk0p7 /data nosuid nodev nodiratime noatime noauto_da_alloc 
-errors=panic
+    mount ext4 /dev/block/mmcblk0p7 /data nosuid nodev nodiratime noatime noauto_da_alloc errors=panic
     mount ext4 /dev/block/mmcblk0p6 /cache nosuid nodev
     mount ext4 /dev/block/mmcblk0p8 /vendor nosuid nodev
-- 
1.8.0

